docs/features.md

```diff
@@ -95,13 +105,26 @@ Some completion items can have documentation associated with them.
 To show the documentation popup you can click the **More** link in the bottom of the autocomplete,
 or you can use the default sublime keybinding `f12` to trigger it.
 
-You can change the default keybinding by remaping the command bellow:
+You can change the default keybinding by remapping the command as below:
 
-```
-{ "keys": ["f12"], "command": "auto_complete_open_link", "context":
-	[
-		{ "key": "auto_complete_visible", "operator": "equal", "operand": true },
-	]
+```js
```

Comment:
The deleted code block meant to remap the `auto_complete_open_link`command to something else.
I imagined that people would just copy this template: 
```
{ "keys": ["f12"], "command": "auto_complete_open_link", "context":
	[
		{ "key": "auto_complete_visible", "operator": "equal", "operand": true },
	]
}
```
paste it in theirs keybinding preferences and remap this `auto_complete_open_link` to something other than `f12`.

Not sure why you deleted this and put a `lsp_symbol_definition` command instead of `auto_complete_open_link` :)

Reply:
Whoops, that should indeed just be `auto_complete_open_link`